### ISSUE TITLE
Download as geo.tiff bug

### ISSUE THREAD

**Issue by cesswairimu** [publiclab/mapknitter#625]
### What happened just before the problem occurred
During the upgrade to rails 4.2, this test was failing for me https://github.com/publiclab/mapknitter/blob/main/test/unit/warpable_test.rb#L62. Thought it was something I broke and been trying to fix it until @alaxalves and @sashadev-sky informed me they were having the same prob with the test. Thanks guys :smiley: 

So I checked out main and tried to investigate more this is the error I got
![Screenshot from 2019-05-21 12-59-08](https://user-images.githubusercontent.com/17081074/58087043-71892600-7bc8-11e9-8824-6ae75cefc73c.png)

### Relevant URL
https://github.com/publiclab/mapknitter/blob/main/app/controllers/export_controller.rb#L30-L32

### Steps to replicate
- On maps show page, click  on the `Export` section.  Hit the `Start Export` button. Make sure the export finishes. 
- On the same page click the `download` drop-down  and select `GeoTIFF`

**Comment by jywarren** [comment]
Ah hmm. Are we sure the geotiff is being generated, and is it possibly just
in the wrong place or wrong filename? We should be able to look into
public/warps/... to check and debug this, after running an export!

**Comment by cesswairimu** [comment]
might be the geotiff its not being created. The folder `public/warps/saugus-landfill-incinerator` exists but `saugus-landfill-incinerator-geo.tif` file is not in the folder. I will need to investigate this further. Thanks Jeff.

**Comment by jywarren** [comment]
Hmm. All the other products are built from the GeoTiff, so if we're not
getting the geotiff we shouldn't get jpg or tiles or zip. I think this is
all working on the live site as i tested it... is this just in your local?
Thanks!

**Comment by cesswairimu** [comment]
Yeah, maybe, could be :thinking:... @alaxalves could you please try to test this locally if you have a minute and let me know if it works fine or otherwise?
